code/game/objects/effects/decals/crayon.dm

```diff
@@ -28,6 +30,20 @@
 	if(main)
 		paint_colour = main
 	add_atom_colour(paint_colour, FIXED_COLOUR_PRIORITY)
-
 /obj/effect/decal/cleanable/crayon/NeverShouldHaveComeHere(turf/T)
 	return isgroundlessturf(T)
+
+/obj/effect/decal/cleanable/crayon/gang

```

Comment:
Why is this a clenable decal?

Reply:
so that you can clean it up same as any other graffiti thing from a spray can?

Reply:
Well that harps backs to the "why is this a special snowflake in is_cleanable".